fem/fe_coll.hpp

```diff
@@ -174,6 +174,27 @@ class FiniteElementCollection
    */
    static FiniteElementCollection *New(const char *name);
 
+   /** Factory method: return a newly allocated H1 conforming FiniteElementCollection
```

Comment:
Although I never use doxygen, I guess we should follow the convention of using `@brief`. Some periods in these comments would also help readability.

Reply:
Added the `@ brief` 4-times.